app/models/report.rb

```diff
@@ -0,0 +1,10 @@
+class Report
+  attr_reader(:year, :month, :cause, :event)
```

Comment:
Method calls that are part of DSLs, or have "keyword status" in Ruby shouldn't use parentheses. 😀